web/src/containers/ScheduleSummary.js

```diff
@@ -16,42 +16,38 @@ const ScheduleSummary = ({ activities }) => (
             {t('scheduleSummary.noDataMessage')}
           </div>
         ) : (
-          activities.map(
-            ({ end, name: activityName, milestones, start }, i) => (
-              <table key={activityName} className="budget-table">
-                <caption className="ds-u-visibility--screen-reader">
-                  Activity {i + 1}: {activityName}
-                </caption>
-                <thead>
-                  <tr>
-                    <th
-                      className="ds-u-font-weight--bold ds-u-border-right--0"
-                      style={{ width: '70%' }}
-                    >
-                      Activity {i + 1}: {activityName}
-                    </th>
-                    <th className="ds-u-font-weight--bold ds-u-padding-right--3 ds-u-text-align--left ds-u-border-left--0 budget-table--cell__nowrap">
-                      {start} – {end}
-                    </th>
-                  </tr>
-                </thead>
-                <tbody>
-                  {milestones.map(
-                    ({ end: milestoneEnd, name: milestoneName }) => (
-                      <tr>
-                        <td className="ds-u-border-right--0">
-                          {milestoneName}
-                        </td>
-                        <td className="ds-u-border-left--0 ds-u-text-align--left">
-                          {milestoneEnd}
-                        </td>
-                      </tr>
-                    )
-                  )}
-                </tbody>
-              </table>
-            )
-          )
+          activities.map(({ name: activityName, dateRange, milestones }, i) => (
```

Comment:
One of the downsides of prettier is that it reformats whole big chunks of code sometimes and ends up changing indentions but nothing else, so it looks like more changed than really did. :sigh: Maybe GH's "hide whitespace" option helps.